src/addons/conditionwatchers/addonconditionwatchertimeend.cpp

```diff
@@ -4,23 +4,24 @@
 
 #include "addonconditionwatchertimeend.h"
 #include "leakdetector.h"
+#include "mfbt/checkedint.h"
 #include "settingsholder.h"
 
 #include <QDateTime>
 
 AddonConditionWatcherTimeEnd::AddonConditionWatcherTimeEnd(QObject* parent,
```

Comment:
Is it possible to merge `AddonConditionWatcherTimeEnd` and `AddonConditionWatcherTimeStart` into a single class? They seem to be mostly redundant.

Reply:
follow up?

Reply:
Sure!

Reply:
VPN-3333